File-Formats/DocIO/Convert-Word-Document-to-Image-Azure.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Convert Word document to Image in Microsoft Azure | Syncfusion
+description: Learn how to convert a Word document to a Image in Azure services using Syncfusion .NET Word (DocIO) library in C#.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to Image in Azure Platform 
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to a Image in Azure services** within a few lines of code.
+
+N> If this is your first time working with Azure, please refer to the dedicated Azure development resources. This section explains how to convert Word documents to Image in C# using the .NET Word (DocIO) library in Azure. 
+
+## Prerequisites 
+* An active **Microsoft Azure subscription** is required. If you don’t have one, please create a free account before starting.
+
+## NuGet Packages Required
```

Comment:
Change it as Azure Services.

Also mention v1 and v4 functions.

Reply:
updated